src/helpers/pivot/spreadsheet_pivot/spreadsheet_pivot.ts

```diff
@@ -396,7 +396,11 @@ export class SpreadsheetPivot implements Pivot<SpreadsheetPivotRuntimeDefinition
     }
     if (
       nonEmptyCells.every(
-        (cell) => cell.type === CellValueType.number && cell.format && isDateTimeFormat(cell.format)
+        (cell) =>
+          cell.type === CellValueType.number &&
+          cell.format &&
+          isDateTimeFormat(cell.format) &&
+          !isDurationFormat(cell.format)
```

Comment:
nitpick: add a check that return `integer` if `isDurationFormat` is true, rather tha relying on the default case